### PR TITLE
proactive-support-bot: Hydra integration for summary/comments

### PR DESCRIPTION
**Pull request by wking**
Builds on #16; review that first.

### CODE REVIEW

File: proactive-support-bot.py [hunk 1]
```diff
@@ -39,6 +42,10 @@ def print_help(self, file=None):
         raise HelpRequest({'parser': self})
 
 
+def dashboard_uri(cluster):
+    return 'https://telemeter-lts-dashboards.datahub.redhat.com/d/kSomLV7Wk/pre-production-cluster-by-id?orgId=1&var-_id={}'.format(cluster)
```
Comment (brenton) [inline proactive-support-bot.py]: probably need to make this configurable.  Also, it leaks an internal hostname.

**Comment by brenton** [comment]
looks good to me overall.

File: proactive-support-bot.py (hunk 1 above)
Comment (wking) [reply proactive-support-bot.py]: We can load it from an env var

Comment (brenton) [reply proactive-support-bot.py]: Yup, something low-tech like that would be fine.

Comment (wking) [reply proactive-support-bot.py]: Fixed with 20d9a7f -> e4ad415, which also rebases onto the new master.

**Pull request merged**